akka-persistence/src/main/scala/akka/persistence/PersistentActor.scala

```diff
@@ -37,47 +45,51 @@ case object RecoveryCompleted extends RecoveryCompleted {
  * @param replayMax maximum number of messages to replay. Default is no limit.
  */
 @SerialVersionUID(1L)
-final case class Recover(fromSnapshot: SnapshotSelectionCriteria = SnapshotSelectionCriteria.Latest, toSequenceNr: Long = Long.MaxValue, replayMax: Long = Long.MaxValue)
```

Comment:
perhaps mention in above docs that it is specified by overriding the `recover` method?